<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="lable_test">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>567</width>
      <height>567</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>-10</y>
      <width>535</width>
      <height>580</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>535</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>535</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_Linear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>170</height>
      </rect>
     </property>
     <property name="title">
      <string>Choose gradient colorspace</string>
     </property>
     <widget class="QRadioButton" name="Rainbow_Colorspace_Radio_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Rainbow</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="BAW_Colorspace_Radio_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Black and White</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RAB_Colorspace_Radio_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>131</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Red and Blue</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Pastel_Colorspace_Radio_Button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Pastel</string>
      </property>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>511</width>
       <height>370</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Linear_Page">
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>371</width>
         <height>160</height>
        </rect>
       </property>
       <property name="title">
        <string>Set Coordinates</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>158</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>X Coordinate, First Point</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="First_X_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>157</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Y Coordinate, First Point</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="First_Y_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>180</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>X Coordinate, Second Point</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Second_X_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>179</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Y Coordinate, Second Point</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Second_Y_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>10</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Continue Shading</string>
       </property>
       <widget class="QCheckBox" name="Continue_Shading_Forward">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>93</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Forward</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>93</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Backward</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>415</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>A linear gradient describes a color change along a straight line</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>415</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>that is defined by two points.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Radial_Page">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>180</y>
         <width>131</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Continue Shading</string>
       </property>
       <widget class="QCheckBox" name="Continue_Shading_Forward_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>93</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Forward</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="Continue_Shading_Backward_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>93</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Backward</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>510</width>
         <height>150</height>
        </rect>
       </property>
       <property name="title">
        <string>Set Coordinates Of Circle Centers</string>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>168</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>X Coordinate, First Center</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="First_Center_X_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>167</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Y Coordinate, First Center</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="First_Center_Y_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>187</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Second Center X Coordinate</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Second_Center_X_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>80</y>
          <width>186</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Second Center Y Coordinate</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Second_Center_Y_Coordinate_Input">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>100</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>371</width>
         <height>100</height>
        </rect>
       </property>
       <property name="title">
        <string>Set Radius Of Circle</string>
       </property>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>75</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>First Radius</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>97</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Second Radius</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="First_Radius_Input">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="Second_Radius_Input">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>319</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Radial gradient describes the color change from</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>300</y>
         <width>335</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>the center to the edges, it is defined by two circles,</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>which are defined by the center and radius</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>117</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Render Gradient</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLinear_Gradient"/>
   <addaction name="actionRadial_Gradient"/>
  </widget>
  <action name="actionLinear_Gradient">
   <property name="text">
    <string>Linear Gradient</string>
   </property>
  </action>
  <action name="actionRadial_Gradient">
   <property name="text">
    <string>Radial Gradient</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>on_button_Push()</signal>
 </slots>
</ui>
